packages/website/src/components/Modals/modal.data.ts

```diff
@@ -0,0 +1,12 @@
+export const forMentors = {
```

Comment:
🙃 To follow our conventions for naming this file should be `Modal.data.ts` + the entire directory name should be `Modal` instead of `Modals`